drivers/pipes/pipe.c

```diff
@@ -207,8 +208,8 @@ static int pipe_register(size_t bufsize, int flags,
 int file_pipe(FAR struct file *filep[2], size_t bufsize, int flags)
 {
   char devname[32];
+  int nonblock = !!(flags & O_NONBLOCK);
```

Comment:
Why not to use bool?

Reply:
Because file_vioctl expect int:
https://github.com/apache/nuttx/blob/master/fs/vfs/fs_ioctl.c#L75
which is mentioned here:
https://man.openbsd.org/ioctl.2